### PR TITLE
chore: copy all samples into samples/snippets

### PR DESCRIPTION
**Pull request by busunkim96**
TODO:

- [ ] pin existing samples to commit hash
- [ ] merge this PR
- [ ] update samples to use master again

### CODE REVIEW

File: noxfile.py [hunk 1]
```diff
@@ -97,28 +97,6 @@ def system(session):
...
-    session.install("grpcio")
-    session.install("-e", ".[all]")
-
-    # Run py.test against the snippets tests.
-    # Skip tests in samples/snippets, as those are run in a different session
-    # using the nox config from that directory.
-    session.run("py.test", os.path.join("docs", "snippets.py"), *session.posargs)
-    session.run("py.test", "samples", "--ignore=samples/snippets", *session.posargs)
-
-
```
Comment (tseaver) [inline noxfile.py:121]: We need to drop the linting of `docs/samples` below (Github isn't letting me leave the comment there).

**Comment by tswast** [comment]
Are we planning on continuing this PR? I notice that we just added a separate Kokoro config for the "snippets".

**Comment by meredithslota** [comment]
I feel like if this task still needs to be done, it might be more efficient to make a new PR — @tswast and @busunkim96 what do you think? cc: @leahecole and @steffnay for context.

**Comment by tswast** [comment]
Yes, we start fresh. I think we can consider this task to be part of https://github.com/googleapis/python-bigquery/issues/366.

**Pull request closed without merging**